docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValuesMultiColumn.cs

```diff
@@ -0,0 +1,119 @@
+                new DataPoint(){ Label = 5, Features1 = new float[3] {-1, 2, -3}, Features2 = new float[2] {-1, float.NaN} },
+                new DataPoint(){ Label = 9, Features1 = new float[3] {-1, 6, -3}, Features2 = new float[2] {0, float.PositiveInfinity} },
+            };
+            // Convert training data to IDataView, the general data type used in ML.NET.
+            var data = mlContext.Data.LoadFromEnumerable(samples);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var meanPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
+            },
+            MissingValueReplacingEstimator.ReplacementMode.Mean);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var meanTransformer = meanPipeline.Fit(data);
+            var meanTransformedData = meanTransformer.Transform(data);
+
+            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
+            var meanRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(meanTransformedData, reuseRowObject: false);
+
+            // ReplaceMissingValues is used to create a column where missing values are replaced according to the ReplacementMode.
+            var defaultPipeline = mlContext.Transforms.ReplaceMissingValues(new[] {
+                new InputOutputColumnPair("MissingReplaced1", "Features1"),
+                new InputOutputColumnPair("MissingReplaced2", "Features2")
+            },
+            MissingValueReplacingEstimator.ReplacementMode.DefaultValue);
+
+            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
+            // This operation doesn't actually evaluate data until we read the data below.
+            var defaultTransformer = defaultPipeline.Fit(data);
+            var defaultTransformedData = defaultTransformer.Transform(data);
+
+            // We can extract the newly created column as an IEnumerable of SampleDataTransformed, the class we define below.
+            var defaultRowEnumerable = mlContext.Data.CreateEnumerable<SampleDataTransformed>(defaultTransformedData, reuseRowObject: false);
+
+            // a small printing utility
+            Func<object[], string> vectorPrinter = (object[] vector) =>
+            {
+                string preview = "[";
+                foreach (var slot in vector)
+                    preview += $"{slot} ";
+                return preview += "]";
+
+            };
```

Comment:
this one can go to the bottom of the file. all the helper utils are secondary, and should go below Example()